pkg/imgpkg/bundle/bundle_test.go

```diff
@@ -82,7 +84,12 @@ func TestPullAllNestedBundlesWritingContentsToDisk(t *testing.T) {
 		defer fakeImagesMetadataBuilder.CleanUp()
 
 		fakeImagesMetadataBuilder.WithBundleFromPath("repo/some-bundle-name", "test_assets/bundle").WithEveryImageFrom("test_assets/image_with_config", map[string]string{})
-		subject := bundle.NewBundle("repo/some-bundle-name", fakeImagesMetadataBuilder.Build())
+
+		build := fakeImagesMetadataBuilder.Build()
+		u, err := url.Parse(fakeImagesMetadataBuilder.server.URL)
```

Comment:
Isn't this equivalent to
`fakeImagesMetadataBuilder.relocateToTestServer("repo/some-bundle-name")`

Reply:
Good catch. Thx I made the change